cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -186,7 +186,7 @@ function run-until-success() {
 # $1 object type
 function get-addons-from-server() {
     local -r obj_type=$1
-    "${KUBECTL}" get "${obj_type}" -o template -t "{{range.items}}{{.metadata.name}} {{end}}" --api-version=v1beta3 -l kubernetes.io/cluster-service=true
```

Comment:
OK, I shall revert both my changes and Brendan's changes and take what is in  #10618 instead.